src/main/java/com/rtb/projectmanagementtool/home/HomeServlet.java

```diff
@@ -16,35 +21,38 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws ServletException, IOException {
 
-    // Mock authentication
-    if (!request.getParameterMap().containsKey("user")) {
-      // Redirect to /login servlet if authentication fails
-      request.getRequestDispatcher("/login").forward(request, response);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    // Authenticate
+    AuthOps auth = new AuthOps(datastore);
+    auth.loginUser(request, response);
+    Long userLoggedInId = auth.whichUserIsLoggedIn(request, response);
```

Comment:
Looks good.  Might be a good idea to encapsulate this in AuthOps as well, maybe something like `getLoggedInUserOrRedirect()`, but I'm not sure how to cleanly handle the early exit yet.